CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+v3.0.7
+ - Fix unstable test test_internet_gateway (#209)
```

Comment:
Do we specify test changes in CHANGELOG? Isn't this document meant to contain feature upgrades/changes?

Reply:
True. But the testing pipeline also runs on the latest release. If we don't release it, the pipeline will keep on failing.

Reply:
Okay, I understand